package.json

```
@@ -39,29 +39,21 @@
     "test:karma": "cross-env NODE_ENV=test TZ=UTC karma start test/karma.conf.js",
     "test:karma:parallel": "cross-env NODE_ENV=test TZ=UTC PARALLEL=true karma start test/karma.conf.js",
     "test:unit": "cross-env NODE_ENV=test TZ=UTC mocha -n expose_gc 'packages/**/*.test.{js,ts,tsx}' 'docs/**/*.test.{js,ts,tsx}'",
-    "test:e2e": "cross-env NODE_ENV=production pnpm test:e2e:build && concurrently --success first --kill-others \"pnpm test:e2e:run\" \"pnpm test:e2e:server\"",
-    "test:e2e:build": "cross-env E2E_BUILD=true webpack --config test/e2e/webpack.config.js",
-    "test:e2e:dev": "concurrently \"pnpm test:e2e:build --watch\" \"pnpm test:e2e:server\"",
-    "test:e2e:run": "mocha --config test/e2e/.mocharc.js 'test/e2e/**/*.test.{js,ts,tsx}'",
-    "test:e2e:server": "serve test/e2e -p 5001",
+    "test:e2e": "pnpm run release:build && cd test/e2e && pnpm run start",
```

Should we use `--filter` instead of `cd` here?
```suggestion
    "test:e2e": "pnpm run release:build && pnpm run --filter test/e2e start",
```

---

You'll need to pass the full package name though:

```suggestion
    "test:e2e": "pnpm run release:build && pnpm run --filter @mui-x-internal/test-e2e start",
```

This will also work:

```suggestion
    "test:e2e": "pnpm run release:build && pnpm run --filter "*test-e2e" start",
```

---

This is more verbose and less clear (imo), I'd leave it as it is.